app/importers/concerns/taxonomy_methods.rb

```diff
@@ -0,0 +1,35 @@
+module TaxonomyMethods
+  attr_accessor :taxonomy_parser
+
+  def get_missing_country(mappable_field)
+    country_array = normalize_industry('Missing Country: ' + mappable_field)
```

Comment:
The idea with this is to map a country onto an entry by sending in a mappable_field, such as a title or description or venue name, that will help someone decide which country should be assigned to the entry.  I prepended the field's value with "Missing Country:  " so that in Mapper, you can keyword search "Missing Country" to see the fields that need a country, and still be able to filter by source (model_name).  It's a different and more specialized use case than the Industry/Topic lookups, but the same functionality.  We could rename the industry lookup methods in IMC and Webservices to something more general.